main.tf

```diff
@@ -0,0 +1,104 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+resource "aws_glue_catalog_database" "this" {
+  name        = var.name
+  description = "Database for analysing ${var.name} with Athena"
+}
+
+data "aws_iam_policy_document" "glue-catalog-database-manage-tables" {
+  statement {
+    actions = [
+      "glue:CreateTable",
+      "glue:UpdateTable",
+      "glue:DeleteTable",
+    ]
+
+    resources = [
+      # https://docs.aws.amazon.com/glue/latest/dg/glue-specifying-resource-arns.html
+      "arn:aws:glue:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:catalog",
+      "arn:aws:glue:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:database/${aws_glue_catalog_database.this.name}",
+      "arn:aws:glue:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:table/${aws_glue_catalog_database.this.name}/*",
+    ]
+  }
+}
+resource "aws_s3_bucket" "athena-workspace" {
```

Comment:
let's add default encryption (as general best practice)

Reply:
@jansiwy I have added service side encryption using the Amazon S3-managed key (with AES256) because the [documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-encryption.html) states the following.

> Only SSE-S3 default encryption is supported for server access log destination buckets.

Reply:
@jtsaito yes that's good, that's also the only option without having to deal with KMS.